src/cache_entry.cc

```diff
@@ -255,7 +255,7 @@ CacheEntry::SerializeResponseOutput(
   // Pack everything into provided buffer
   uint64_t position = 0;
```

Comment:
out of curiosity, why this line doesn't cause issues

Reply:
@oandreeva-nv you have a good point. I'll elaborate a bit more on this one. The real issue is with the following lines. SerializeResponseOutput is expecting a size_t pointer:
`CacheEntry::SerializeResponseOutput(
    const InferenceResponse::Output& output, Byte* buffer, size_t* output_size)`

For our platform there is a mismatch between these two types:
error: cannot convert ‘uint64_t*’ {aka ‘long long unsigned int*’} to ‘size_t*’ {aka ‘unsigned int*’}